compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/SingleLayerComposeScene.skiko.kt

```diff
@@ -171,7 +172,7 @@ private class SingleLayerComposeSceneImpl(
         mainOwner.measureAndLayout()
     }
 
-    override fun draw(canvas: Canvas) {
+    override fun draw(canvas: Canvas) = trace("SingleLayerComposeSceneImpl:draw") {
```

Comment:
Move it to owner, no need to log "SingleLayerComposeSceneImpl"

Reply:
Done